code/style.css

```diff
@@ -1,58 +1,264 @@
 body {
   margin: 0;
   font-family: "Courier New", Courier, monospace;
```

Comment:
Maybe you can set this to the Font you used the most and change it in specific places where you use another font?